### ISSUE TITLE
Function round

### ISSUE THREAD

**Issue by mg901** [oddbird/accoutrement-scale#8]
Miriam, good time of day! Tell me please, is it possible to use the sass function round directly in $ sizes? If it is possible, then how can this be done? Please show the example of 'xs-leading'.
Thanks in advance.
`$ratios: (
  'major-third': 1.25,
  'minor-sixth': 1.6,
);

$sizes: (
  'root': 16px,
  'xs-line': 1 ('minor-sixth': 1),

  // XS
  'xs-base': 16px,
  'xs-leading': 'xs-base' ('minor-sixth': 1),

  'xs-text': 'xs-base' ('major-third': 0, 'convert-units': 'rem'),
  'xs-h1': 'xs-base' ('major-third': 5, 'convert-units': 'rem'),
  'xs-h2': 'xs-base' ('major-third': 4, 'convert-units': 'rem'),
  'xs-h3': 'xs-base' ('major-third': 3, 'convert-units': 'rem'),
  'xs-h4': 'xs-base' ('major-third': 2, 'convert-units': 'rem'),
  'xs-h5': 'xs-base' ('major-third': 1, 'convert-units': 'rem'),
  'xs-h6': 'xs-base' ('major-third': 1, 'convert-units': 'rem'),
)
`

**Comment by mg901** [comment]
I apologize. I looked at the source code and understood that there is no such possibility.

**Comment by mirisuzanne** [comment]
Actually, I think this would be solvable, and a nice feature. You should be able to:

```
'xs-leading': 'xs-base' ('minor-sixth': 1, 'round': 'no-args'),
```

I just made up `no-args`, and maybe we could come up with something better. The only reason it doesn't work currently is that it assumes you always have at least one argument to pass along.

**Comment by mirisuzanne** [comment]
closed by 9710a0b
